packages/ember-glimmer/tests/utils/shared-conditional-tests.js

```diff
@@ -154,6 +151,19 @@ export const BASIC_FALSY_TESTS = {
 
 export class SharedConditionalsTest extends AbstractConditionalsTest {
```

Comment:
What is the difference between `SharedConditionalsTest` and `AbstractConditionalsTest`?